config/packages/kimai.yaml

```
@@ -97,38 +97,74 @@ kimai:
     # Language specific settings, like the date formats
     languages:
         de:
+            date_time: 'dd.MM.yyyy HH:mm'
+            date_time_picker: 'DD.MM.YYYY HH:mm'
+            date: 'dd.MM.yyyy'
+            date_picker: 'DD.MM.YYYY'
             date_short: 'd.m.Y'
             duration: '%%h:%%m Stunden'
             duration_short: '%%h:%%m h'
         en:
+            date_time: 'yyyy-MM-dd HH:mm'
+            date_time_picker: 'YYYY-MM-DD HH:mm'
+            date: 'yyyy-MM-dd'
+            date_picker: 'YYYY-MM-DD'
             date_short: 'Y-m-d'
             duration: '%%h:%%m hours'
             duration_short: '%%h:%%m h'
         pt_BR:
+            date_time: 'dd-MM-yyyy HH:mm'
+            date_time_picker: 'DD-MM-YYYY HH:mm'
+            date: 'dd-MM-yyyy'
+            date_picker: 'DD-MM-YYYY'
             date_short: 'd-m-Y'
             duration: '%%h:%%m horas'
             duration_short: '%%h:%%m h'
         it:
+            date_time: 'dd.MM.yyyy HH:mm'
+            date_time_picker: 'DD.MM.YYYY HH:mm'
+            date: 'dd.MM.yyyy'
+            date_picker: 'DD.MM.YYYY'
             date_short: 'd.m.Y'
             duration: '%%h:%%m h'
             duration_short: '%%h:%%m h'
         fr:
+            date_time: 'dd/MM/yyyy HH:mm'
+            date_time_picker: 'DD/MM/YYYY HH:mm'
+            date: 'dd/MM/yyyy'
+            date_picker: 'DD/MM/YYYY'
             date_short: 'd/m/Y'
             duration: '%%h h %%m min'
             duration_short: '%%h h %%m'
         es:
+            date_time: 'dd.MM.yyyy HH:mm'
+            date_time_picker: 'DD.MM.YYYY HH:mm'
+            date: 'dd.MM.yyyy'
+            date_picker: 'DD.MM.YYYY'
             date_short: 'd.m.Y'
             duration: '%%h:%%m h'
             duration_short: '%%h:%%m h'
         ru:
+            date_time: 'dd.MM.yyyy HH:mm'
+            date_time_picker: 'DD.MM.YYYY HH:mm'
+            date: 'dd.MM.yyyy'
+            date_picker: 'DD.MM.YYYY'
             date_short: 'd.m.Y'
             duration: '%%h:%%m Часов'
             duration_short: '%%h:%%m h'
         ar:
+            date_time: 'yyyy-MM-dd HH:mm'
+            date_time_picker: 'YYYY-MM-DD HH:mm'
+            date: 'yyyy-MM-dd'
+            date_picker: 'YYYY-MM-DD'
             date_short: 'Y-m-d'
             duration: '%%h:%%m hours'
             duration_short: '%%h:%%m h'
         hu:
+            date_time: 'yyyy.MM.dd HH:mm'
+            date_time_picker: 'YYYY.MM.DD HH:mm'
+            date: 'yyyy.MM.dd'
+            date_picker: 'YYYY.MM.DD'
```

```suggestion
            ddate_time: 'yyyy.MM.dd. HH:mm'
            date_time_picker: 'YYYY.MM.DD. HH:mm'
            date: 'yyyy.MM.dd.'
            date_picker: 'YYYY.MM.DD.'
```
In Hungarian we always put a dot after the number of the day. 
If you don't fix it now, I will fix it when I update the translation with the new words.

---

There is a typo in `ddate_time` so I can't commit it directly here.